pkg/process/util/ss_bytes.go

```diff
@@ -0,0 +1,36 @@
+package util
+
+import (
+	"bytes"
+	"sort"
+)
+
+type SSBytes [][]byte
+
+var _ sort.Interface = SSBytes{}
+
+func (ss SSBytes) Len() int {
+	return len(ss)
+}
+
+func (ss SSBytes) Less(i, j int) bool {
+	return bytes.Compare(ss[i], ss[j]) < 0
+}
+
+func (ss SSBytes) Swap(i, j int) {
+	ss[i], ss[j] = ss[j], ss[i]
+}
+
+// Search returns the index of element x if found or the length of the SSBytes otherwise.
+// SSBytes is expected to be sorted.
+func (ss SSBytes) Search(x []byte) int {
+	i := sort.Search(len(ss), func(i int) bool {
```

Comment:
There is a `sort.SearchStrings` which may simplify this code quite a bit.

Reply:
That would require you to allocate strings